src/css/_scrollbar.scss

```diff
@@ -7,7 +7,7 @@
   ::-webkit-scrollbar {
     width: 17px;
     height: 18px;
-    border-left: 0.5px solid var(--sn-stylekit-scrollbar-track-border-color);
```

Comment:
I think ‘none’ is the way to go?

Reply:
Changed in https://github.com/standardnotes/StyleKit/pull/22/commits/08e452753066f8f5eb97c49d6a1c0371e63565e8

Reply:
I think `border-left: none;` is the default. Is it better to delete this line or specify `border-left: none;`?

Reply:
Would depend on how sure you are 😄 Not a huge deal either way.